sol/private/sol_binary.bzl

```diff
@@ -140,8 +144,22 @@ def _run_solc(ctx):
 def _sol_binary_impl(ctx):
     return _run_solc(ctx)
 
+def _impl(settings, attr):
```

Comment:
`_solc_version_transition_impl` as `_impl` may become confusing when not in the context of this PR.